app/assets/scripts/map.js

```diff
@@ -14,7 +15,7 @@ var infowindow;
 function initMap() {
   map = new google.maps.Map(document.getElementById('map'), {
     center: {lat: 39.949, lng: -75.181}, // TODO(#52): Do not hardcode this.
-    zoom: 13
+    zoom: focusZoom,
```

Comment:
Yeah I'm thinking of just having this default value? and otherwise I'll remove the focus and zoom like you did in your PR and have that automatically adjust?